src/Illuminate/Notifications/Channels/SlackWebhookChannel.php

```diff
@@ -111,4 +114,21 @@ protected function fields(SlackAttachment $attachment)
             return ['title' => $key, 'value' => $value, 'short' => true];
         })->values()->all();
     }
+
+    /**
+     * Format the attachment's actions.
+     *
+     * @param  \Illuminate\Notifications\Messages\SlackAttachment  $attachment
+     * @return array
+     */
+    protected function actions(SlackAttachment $attachment)
+    {
+        return collect($attachment->actions)->map(function ($value) {
```

Comment:
Makes sense to me 👍 I'd just gone along with formatting similar to some related Slack Notification methods. This seems simpler and is likely faster. Thanks @lucasmichot!